views/forms/TextInput/index.jsx

```diff
@@ -20,6 +20,7 @@ const TextInput = (props) => {
       type={props.type}
       placeholder={props.placeholder}
       sx={props.sx}
+      disabled={props.disabled}
```

Comment:
If we're passing through `props` can we use `...props` instead?

Reply:
yeah, I think we want to be more deliberate with these props if we spread the props we have a chance of adding a prop that may not be present in the component